dev-python/h5py/h5py-2.10.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 flag-o-matic
+
+DESCRIPTION="Simple Python interface to HDF5 files"
+HOMEPAGE="https://github.com/h5py/h5py https://www.h5py.org"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test examples mpi"
+
+RDEPEND="
+	sci-libs/hdf5:=[mpi=,hl(+)]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]"
+
+BDEPEND="dev-python/pkgconfig"
+
+DEPEND="${RDEPEND}
+	dev-python/cython[${PYTHON_USEDEP}]
+	mpi? ( dev-python/mpi4py[${PYTHON_USEDEP}] )
+	doc? ( dev-python/alabaster[${PYTHON_USEDEP}] )
```

Comment:
Can this be added to `distutils_enable_sphinx` how it should work?

Reply:
No, sadly that's not possible. When using the `--no-autodoc` option, you can't specify additional dependencies. And `doc` only works with the `--no-autodoc` option in this package, (I think that that has something to do with the upstream conf.py not specifiying some things, but I don't know for sure).